src/controllers/manage/list/appointmentsListViewController.js

```diff
@@ -72,6 +99,23 @@ angular.module('bahmni.appointments')
                 });
             };
 
+            var modifyAppointmentPriorities = function(){
+                $scope.filteredAppointments.map((appointment) => {
+                    var priorityModified = false;
+                    appConfig.config.priorityOptionsList.map((priority) => {
```

Comment:
@Arjun-Go what if we don't configure appConfig.config.priorityOptionsList ?

Reply:
@binduak In a scenario where `priorityOptionList` is not present/configured, we would face a broken UI with an undefined or null value error. Now for CURE, we have it configured already [here](https://github.com/cureinternational/cure-bahmni-emr/blob/7324ac3a08e1ea46956284b0ef52cbe66a13769d/openmrs/apps/appointments/app.json#L24). 
But when it comes to Bahmni, we would have a broken UI, I agree.

Reply:
The code needs to be backward compatible, assuming that people are upgrading to new docker images, but using their older "config" folders.

Reply:
Have raised a thread in bahmni-ipd channel for the same - https://bahmni.slack.com/archives/C056DPYNDN3/p1684755171955239

Reply:
@gsluthra we need this PR for internal showcase. We will continue the discussion in the #bahmni-ipd channel and will work on as per the inputs from product
on creating JIRA cards for the product code changes, we would like to it take as part of tomorrows architecture call to come up with the process. Hope that is fine.